<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find Out Palindrome String</string>
  </property>
  <property name="windowIcon">
   <iconset resource="iconbutton.qrc">
    <normaloff>:/mFZ7zm1w.png</normaloff>:/mFZ7zm1w.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="2" column="0">
       <widget class="QProgressBar" name="ProgressBar">
        <property name="styleSheet">
         <string notr="true">QProgressBar {
     border: 2px solid grey;
     border-radius: 4px;
     background-color: #993300;
 }

 QProgressBar::chunk {
     background-color: #666633;
     width: 20px;
 }</string>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="format">
         <string>Ready !</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGraphicsView" name="Shower">
        <property name="styleSheet">
         <string notr="true">QGraphicsView{
     border: 2px solid grey;
     border-radius: 4px;
     background-color: #000000;
 }</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTextEdit" name="ShowResult">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QTextEdit {
     border: 2px solid grey;
     border-radius: 4px;
     background-color: #993300;
 }</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="midLineWidth">
         <number>1</number>
        </property>
        <property name="undoRedoEnabled">
         <bool>false</bool>
        </property>
        <property name="lineWrapMode">
         <enum>QTextEdit::WidgetWidth</enum>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>171</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="title">
         <string>Status:</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <widget class="QLCDNumber" name="InputLengh">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLCDNumber {
     border: 2px solid grey;
     border-radius: 4px;
     color: #ef0707;
 }</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="PalindromeLengh">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>40</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLCDNumber {
     border: 2px solid grey;
     border-radius: 4px;
     color: #ef0707;
 }</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Input Lengh:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>20</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>P-Lengh:</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="ItemsCount">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLCDNumber {
     border: 2px solid grey;
     border-radius: 4px;
     color: #ef0707;
 }</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Items:</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="MemnoryCount">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>90</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLCDNumber {
     border: 2px solid grey;
     border-radius: 4px;
     color: #ef0707;
 }</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>70</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>M-Bytes:</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="Waiting">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>140</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLCDNumber {
     border: 2px solid grey;
     border-radius: 4px;
     color: #ef0707;
 }</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="Done">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>140</y>
           <width>81</width>
           <height>31</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QLCDNumber {
     border: 2px solid grey;
     border-radius: 4px;
     color: #ef0707;
 }</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Waiting:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>120</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Completed:</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>90</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Input/Output:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <widget class="QToolButton" name="InputKeyBoard">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>From Keyboard</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="LinkInput">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>151</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string>Read string from file</string>
         </property>
         <property name="text">
          <string>Load from file</string>
         </property>
        </widget>
        <widget class="QPushButton" name="OpenFile">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>30</y>
           <width>31</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>7</pointsize>
          </font>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhNone</set>
         </property>
         <property name="icon">
          <iconset resource="iconbutton.qrc">
           <normaloff>:/Open_file.png</normaloff>:/Open_file.png</iconset>
         </property>
        </widget>
        <widget class="QTextEdit" name="TextInput">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>181</width>
           <height>71</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
        <widget class="QToolButton" name="SaveToFile">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>50</y>
           <width>31</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>7</pointsize>
          </font>
         </property>
         <property name="icon">
          <iconset resource="iconbutton.qrc">
           <normaloff>:/Save_file</normaloff>:/Save_file</iconset>
         </property>
        </widget>
        <widget class="QLineEdit" name="LinkOutput">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>151</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Save to file</string>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>82</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>81</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="title">
         <string>Control:</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <widget class="QToolButton" name="StartDemo">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Normal Run</string>
         </property>
        </widget>
        <widget class="QToolButton" name="QuickFindDown">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Quick Run</string>
         </property>
        </widget>
        <widget class="QToolButton" name="ResetAll">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>81</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
        <widget class="QSlider" name="SpeedDemo">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>20</y>
           <width>22</width>
           <height>61</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QSlider {
     border: 1px solid grey;
     color: #4b42f4;
 }</string>
         </property>
         <property name="minimum">
          <number>100</number>
         </property>
         <property name="maximum">
          <number>2000</number>
         </property>
         <property name="sliderPosition">
          <number>1500</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>30</y>
           <width>51</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Speed:</string>
         </property>
        </widget>
        <widget class="QLCDNumber" name="SpeedShow">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>50</y>
           <width>64</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLCDNumber {
     border: 2px solid grey;
     border-radius: 4px;
     color: #4b42f4;
 }</string>
         </property>
         <property name="smallDecimalPoint">
          <bool>false</bool>
         </property>
         <property name="intValue" stdset="0">
          <number>1500</number>
         </property>
        </widget>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Note:</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <widget class="QGraphicsView" name="cmd">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>201</width>
           <height>41</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QGraphicsView{
     border: 2px solid grey;
     border-radius: 4px;
     background-color: #000000;
 }</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>290</y>
           <width>191</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Product by: B1400483-CTU/CIT</string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="iconbutton.qrc"/>
 </resources>
 <connections/>
</ui>
